addons/website/static/src/snippets/s_table_of_content/000.js

```diff
@@ -13,6 +13,11 @@ const TableOfContent = publicWidget.Widget.extend({
      */
     async start() {
         this._stripNavbarStyles();
+
+        // Those links will be recreated when entering edit mode, see
+        // `_generateNav`. No need to recreate those.
+        this.$target.find('.table_of_content_link:hidden').remove();
+
```

Comment:
In version 16.0, we have upgraded to Bootstrap 5.1.3. In version 15.0, we were using Bootstrap 4.1.3, which caused an issue where the `0px` CSS rule was overriding the `-1px` rule for the last item when the last element was invisible. However, in Bootstrap 5.1.3, this issue has been resolved, and this specific case is no longer generated. As a result, we can safely remove the lines of code that were added to address this issue. For more information on why these changes were made, please refer [here.](https://github.com/odoo/odoo/pull/126705#discussion_r1301760101)